ui/apps/platform/src/utils/dateUtils.ts

```diff
@@ -165,6 +165,13 @@ export function getDayOfMonthWithOrdinal(num: number): string {
     }
 }
 
+export const formatDate = (date: Date) => {
+    const day = `0${date.getDate()}`.slice(-2);
+    const month = `0${date.getMonth() + 1}`.slice(-2);
+    const year = date.getFullYear();
+    return `${month}/${day}/${year}`;
+};
+
```

Comment:
Could you use the ` getDate` function above instead of doing the formatting manually?

Also, could we rename this to something like `formatDateForApiSearch`?

Reply:
Ah, you're right. We don't even need this function. I'll use `getDate`